spec/forms/two_factor_options_form_spec.rb

```diff
@@ -111,7 +111,7 @@
     end
 
     context 'when a user wants to is required to add piv_cac on sign in' do
-      let(:user) { build(:user, :with_authentication_app) }
+      let(:user) { build(:user) }
```

Comment:
Why was this change necessary?

Reply:
I think I was stuck on something and removing with_authentication_app relieved it. It doesn't appear to be causing issues now with it added back.